cfme/tests/cloud/test_providers.py

```diff
@@ -36,6 +41,84 @@ def enable_regions(provider):
     enable_provider_regions(provider)
 
 
+@pytest.fixture(scope='module')
+def az_pwsh_vm():
+    # azure_pwsh contains powershell and necessary scripts
+    provider = get_crud_by_name('rhv42')
+    pwsh_vm = provider.data.powershell_vm
+    collection = provider.appliance.provider_based_collection(provider)
+    try:
+        return collection.instantiate(pwsh_vm, provider)
+    except IndexError:
+        from cfme.exceptions import ItemNotFound
+        raise ItemNotFound('VM with powershell not found!')
```

Comment:
This should be a `pytest.skip` call, we don't have the resource necessary to run the test.